R/doeFull.R

```diff
@@ -16,92 +16,99 @@
 #
 
 #' @export
-doeFull <- function(jaspResults, dataset, options, ...){
+doeFull <- function(jaspResults, dataset, options, ...) {
+  if (options[["setSeed"]]) {
+    set.seed(options[["seed"]])
+  }
```

Comment:
```suggestion
  jaspBase::.setSeedJASP(options)
```
see https://github.com/jasp-stats/jaspBase/blob/e7b6ed5d260aefcbd562d3d382af41e116ff0735/R/common.R#L625

Reply:
This file doeFull.R is not used anymore actually, both analyses are merged in doeFactorial.R now. These changes are from reformatting old code a while ago I think